src/Quadrature/quadrature.jl

```diff
@@ -100,10 +100,10 @@ julia> getpoints(qr)
 """
 getpoints(qr::QuadratureRule) = qr.points
 
-(::Type{QuadratureRule{dim, shape}}){dim, shape}(order::Int) = QuadratureRule{dim, shape}(:legendre, order)
+QuadratureRule{dim, shape}(order::Int) where {dim, shape} = QuadratureRule{dim, shape}(:legendre, order)
 
 # Special case for face integration of 1D problems
-function (::Type{QuadratureRule{0, RefCube}})(quad_type::Symbol, order::Int)
+function QuadratureRule{0, RefCube}(quad_type::Symbol, order::Int)
```

Comment:
Isn't this exactly what we do here? https://github.com/KristofferC/Tensors.jl/blob/b72cad701c5007a417a882f939dfe6bb2616f864/src/Tensors.jl#L96